NuGet.Config

```diff
@@ -73,6 +72,7 @@
       <package pattern="Microsoft.VisualStudio.*" />
       <package pattern="Microsoft.VisualStudioEng.MicroBuild.Core" />
       <package pattern="Microsoft.VSSDK.BuildTools" />
+      <package pattern="nerdbank.streams" />
```

Comment:
Was this version not available on this feed, I'm guessing?

Reply:
Correct. The version that's a dependency of ServiceHub is not in that feed but is in the vside feed.

Reply:
I'm working on getting the version that ServiceHub needs into dotnet-public and I'll put this package source mapping back. I'll push an update when that's ready.

Reply:
This is addressed now in the latest commit.